spec/requests/users_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Users API', type: :request do
+  let(:valid_user_params) do
+    {
+      user: {
+        name: Faker::Name.name,
+        email: Faker::Internet.email,
+        password: 'Password123+',
+        password_confirmation: 'Password123+'
+      }
+    }
+  end
+
+  describe 'Signup' do
+    let(:headers) do
```

Comment:
This can be replaced with 1 line `let(:headers) { { 'ACCEPT' => 'application/json'}  }

Reply:
Rubocop asks to put let blocks with do;end rather then the brackets.
I'll try to inline it with do;end; 
Is this cop necessary?

Reply:
Lets disable that cop then

Reply:
I'll be more than happy to do it HH